lib/dynamo/dynamo/util.py

```diff
@@ -36,3 +36,13 @@ def convert_floats_to_decimals(element):
     if type(element) is dict:
         return {key: convert_floats_to_decimals(value) for key, value in element.items()}
     return element
+
+
+def convert_decimals_to_floats(element):
+    if type(element) is Decimal:
+        return float(element)
```

Comment:
I suspect there are some values we would want to cast as `int` instead of float? I expect this issue was uncovered because a subscription included path/frame search parameters, which I would expect to pass to asf_search as `int`s?

Reply:
`asf_search.search` function definition: https://github.com/asfadmin/Discovery-asf_search/blob/master/asf_search/search/search.py#L39